terranodo/eventkit-cloud
# 23023 loading icons

This PR adds progress icons to the Status & Download page, DataPackPage, and Login loading page.
User will probably not see the login loading page since it its usually so quick.
Users should see a progress circle while waiting for datapacks to load or be deleted on the DataPack page.
Users should see a progress circle while waiting for run information to load or run to be deleted on the Status & Download page. 
(Also removed css from login pages)

![image](https://cloud.githubusercontent.com/assets/16799449/26729366/6cab2abc-477b-11e7-964d-90caf9f71d9d.png)
![image](https://cloud.githubusercontent.com/assets/16799449/26729499/cc1fe4f6-477b-11e7-8fcb-efe1a6f89f70.png)
![image](https://cloud.githubusercontent.com/assets/16799449/26729626/30b55e28-477c-11e7-991c-19ed990ba399.png)
![image](https://cloud.githubusercontent.com/assets/16799449/26729771/b2cda47e-477c-11e7-8477-5ba282dc08b7.png)

To test:
Using s3 can delay the deleting enough to see the icon, but for initial loading/login you may need to wrap axios calls in a timeout so you have time to see the icon.
